### ISSUE TITLE
proxy mode: add support for disabling head insert and banner

### ISSUE THREAD

**Issue by ikreymer** [webrecorder/pywb#316]
Proxy mode does not include `wombat.js`, however, the head insert is still added by default and the banner is still displayed. Add the following options, which default to true.

```
proxy:
    ...
    use_banner: true
    use_head_insert: true
```

- If `use_banner` is set to false, the banner in `banner.html` will not be added. (A custom banner can override this property, which is checked in the Jinja2 template)

- If `use_head_insert` is false, no modifications are made to the page, `head_insert.html` is not inserted at all.

**Comment by ikreymer** [comment]
@despens does this address all of your use cases also?

**Comment by ikreymer** [comment]
> Is this possible to set per collection?

No, since the proxy settings are global currently and there can only be one collection set for proxy mode, eg.
```
proxy:
    coll: my-coll
    use_banner: true
    use_head_insert: true
```

Not sure it makes sense per-collection, unless there is a way to configure multiple proxy mode collections in base pywb.

> Is true the default for both user_banner and use_head_insert?

Yes, this was done for consistency. I think this depends, if proxy mode access is provided publicly, then probably a good idea to include banner to avoid confusion. If within a remote browser, then its not needed.

**Comment by ikreymer** [comment]
Released in 2.0.4
